api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -350,14 +350,16 @@ public Role saveRole(Role role) throws APIException {
 	 */
 	@Deprecated
 	public void changePassword(User u, String pw) throws APIException {
-		dao.changePassword(u, pw);
+		changePassword(u, "test", pw);
```

Comment:
This method was a mistake in the first place, i guess you have to validate the new password manually and keep the old code for now, it is going to be removed soon since it is deprecated. Sorry about that!